site/src/pages/TaskPage/TaskPage.tsx

```diff
@@ -174,6 +168,84 @@ const TaskPage = () => {
 
 export default TaskPage;
 
+type WorkspaceNotRunningProps = {
+	task: Task;
+};
+
+const WorkspaceNotRunning: FC<WorkspaceNotRunningProps> = ({ task }) => {
+	const queryClient = useQueryClient();
+
+	const { data: parameters } = useQuery({
+		queryKey: ["workspace", task.workspace.id, "parameters"],
+		queryFn: () => API.getWorkspaceParameters(task.workspace),
+	});
+
+	const [workspaceErrorDialog, setWorkspaceErrorDialog] = useState<{
+		open: boolean;
+		error?: ApiError;
+	}>({ open: false });
+
+	const handleError = (error: unknown) => {
+		if (isApiError(error)) {
+			setWorkspaceErrorDialog({
+				open: true,
+				error: error,
```

Comment:
I don't think you need to store this in state, you can just access `mutateStartWorkspace.error`